R/every-some.R

```diff
@@ -26,10 +26,12 @@ every <- function(.x, .p, ...) {
 #' @rdname every
 some <- function(.x, .p, ...) {
   .p <- as_mapper(.p, ...)
+  no_na <- TRUE
```

Comment:
What if we did something like this?

```R
val <- NA
...

val <- val || .p(.x[[i]], ...)

...

return(val)
```

(let me know if I've skipped too many steps so this doesn't make sense)